go/fn/object_test.go

```diff
@@ -213,6 +213,49 @@ metadata:
 	}
 }
 
+func TestSetSlice(t *testing.T) {
+	var original = []byte(`apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: write-pods
+  namespace: default
+subjects:
+- kind: User
+  apiGroup: testing.group
+`)
+
+	var other = []byte(`apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-pods
+  namespace: default
+subjects:
+- kind: User
+  apiGroup: rbac.authorization.k8s.io
+- kind: Admin
+  apiGroup: rbac.authorization.k8s.io
+`)
+
+	var expected = `apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: write-pods
+  namespace: default
+subjects:
+- kind: User
+  apiGroup: rbac.authorization.k8s.io
+- kind: Admin
+  apiGroup: rbac.authorization.k8s.io
+`
+	originalObj, _ := ParseKubeObject(original)
```

Comment:
I don't think we should ignore these errors, here and in the next line. We can do a similar thing as I suggested in my other comment to use the `require` ("github.com/stretchr/testify/require") package:

```
originalObj, err := ParseKubeObject(original)
require.NoError(t, err)
```